Docker/tracer/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# run as:
```

Comment:
I'm not a big fan of these containers either... but refactoring them into a recipe seems a huge pain that I would like to avoid. The dependencies seem really fragile and the official images at least work. The trouble already starts that according to their Readme, the python package versions of bracer and tracer are not compatible :/